### PR TITLE
fix(send): FeeLevels reversed order (Low > High)

### LINKED ISSUE #2331: Fee: Low to Custom
Order of the fee types should be _Low – Medium – High – Custom_ ([as designed](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/5efc87aefaf0aa9cd861d17f))

### PR DESCRIPTION
**Pull request by szymonlesisz**
fix: #2331

### CODE REVIEW

File: packages/suite/src/actions/wallet/blockchainActions.ts [hunk 1]
```diff
@@ -38,6 +39,15 @@ export type BlockchainActions =
           payload: Partial<FeeState>;
       };
 
+// sort FeeLevels in reversed order (Low > High)
+// TODO: consider to use same order in trezor-connect to avoid double sorting
```
Comment (keraf) [inline packages/suite/src/actions/wallet/blockchainActions.ts:43]: Good point, we could add this for 8.1.15.

**Review by keraf — APPROVED** [review]
Looks good, let's maybe get it changed in trezor-connect as you mentioned in your todo.

**Comment by szymonlesisz** [comment]
i mean, i'm not sure this change in trezor-connect doesn't break anything. that's why i decided not to put it there yet and think about it

**Comment by keraf** [comment]
Yeah I'm a bit worried about this too. Let's merge it for now and we'll think about it.

**Pull request merged**